modules/profile/files/channelmgnt.json

```diff
@@ -173,4 +173,15 @@
          "Universal_Omega"
       ]
    }
```

Comment:
There also needs to be a comma at the end of this line

Reply:
@MacFan4000 Oh, I copied the `#miraheze-feed-meta`, and there was no comma. Should I fix that as well?

Reply:
No, JSON syntax means have a comma for all but the last item

Reply:
@MacFan4000 Oh, you mean last item in the file, not last item in the specific array?

Reply:
No, in the array

Reply:
@MacFan4000 Okay, I'm confused then because I have commas after every line except the last one, don't I?

Reply:
What I’m saying is the the } here needs to change to },

Reply:
@MacFan4000 Ah, okay, thanks. Sorry for the confusion. Doing now.